### PR TITLE
Switch containers to Fedora with pulp-rpm

### PR DESCRIPTION
**Pull request by mikedep333**
Also includes some other fixes/improvements that are related, or were related during development/troubleshooting.

fixes: #5176
https://pulp.plan.io/issues/5176
"Port pulp containers to Fedora 30"

Required PR: https://github.com/pulp/pulp-operator/pull/10

### CODE REVIEW

File: containers/images/pulp/Dockerfile [hunk 1]
```diff
@@ -1,41 +1,74 @@
...
+# glibc subpackages like glibc-langpack-en.
+RUN rm /etc/rpm/macros.image-language-conf
+RUN echo 'LANG="en_US.UTF-8"' > /etc/locale.conf
 ENV LANG=en_US.UTF-8
 ENV LANGUAGE=en_US.UTF-8
 ENV LC_ALL=en_US.UTF-8
 ENV PYTHONUNBUFFERED=0
 ENV DJANGO_SETTINGS_MODULE=pulpcore.app.settings
 
+# mariadb_devel needed to avoid error installing pulpcore[mysql] (post-rc1)
```
Comment (dkliban) [inline containers/images/pulp/Dockerfile]: we don't need mariadb anymore.

Comment (mikedep333) [reply containers/images/pulp/Dockerfile]: I was wondering if I can finally clean this up or not. I'll remove it.

Comment (mikedep333) [reply containers/images/pulp/Dockerfile]: I just rebuilt the containers (still using nightly/master pulpcore via pip calling git, not the branch of the PR) and now I'm getting this error that was discussed earlier today:
https://travis-ci.com/pulp/pulp-operator/jobs/220638002#L944

Comment (dkliban) [reply containers/images/pulp/Dockerfile]: merging https://github.com/pulp/pulp_rpm/pull/1413 will fix the problem.

Comment (mikedep333) [reply containers/images/pulp/Dockerfile]: @dkliban It's such a simple PR. I'll just wait for it to be merged, and switch to the nightly versions of plugins. (Or let pulp_rpm do a release and stick with stable releases of plugins.)

**Comment by mikedep333** [comment]
@dkliban pulp-operator tests passed. Please merge:
https://travis-ci.com/pulp/pulp-operator/builds/121151336

Note: I had to build the images with the nightly content plugins for drf 3.10 compatibility (cookbook was updated yesterday evening, moments before I repeated the build).

File: containers/images/pulp/Dockerfile (hunk 1 above)
Comment (mikedep333) [reply containers/images/pulp/Dockerfile]: I made the change (before running the pulp-operator tests.)

**Pull request merged**